.tekton/telco-core-rds-4-19-pull-request.yaml

```diff
@@ -105,7 +105,7 @@ spec:
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
+    - default: "true"
```

Comment:
why? you are already overriding that when you passed it as "true":
https://github.com/openshift-kni/telco-reference/pull/120/commits/7b007c997009f797de355414a2deee04c8ef9c29#diff-9b1c508ae64c738b5dd0095fcce563c6f9c2138e90e1c119657a0872d0741276R37